### PR TITLE
ADEM-7268 MBLB waits for recirculation-prefooter

### PR DESCRIPTION
**Pull request by leshniak**
## Links

https://wikia-inc.atlassian.net/browse/ADEN-7134

## Description

Due to low viewability for BLB, we forced an ad-slot to inject after the recirculation-prefooter renders.

## Reviewers

@nandy-andy 
@bkoval

### CODE REVIEW

**Review by nandy-andy — APPROVED** [review]

File: app/components/recirculation-prefooter.js [hunk 1]
```diff
@@ -51,6 +55,9 @@ export default Component.extend(
 				},
 				intersectionThreshold: 0
 			});
+
+			this.listRendered = defer();
```
Comment (leshniak) [inline app/components/recirculation-prefooter.js]: I know that's not the prettiest solution 😛

File: app/components/recirculation-prefooter.js [hunk 2]
```diff
@@ -49,8 +53,11 @@ export default Component.extend(
 					left: 0,
 					right: 0
 				},
-				intersectionThreshold: 0
+				intersectionThreshold: 0,
+				listRendered: defer()
```
Comment (leshniak) [inline app/components/recirculation-prefooter.js:57]: I know that's not the prettiest solution 😛

**Comment by bkoval** [comment]
How about using Ember.Evented + a service to make that look a bit less untangled? https://emberigniter.com/communication-between-distant-components/ We can discuss it offline, but I think it can make the code a bit more maintainable for you :)

**Comment by nandy-andy** [comment]
@bkoval we decided to go with this solution as it's more flexible and we need to fix the issue ASAP. Feel free to join us in designing a more Ember-like solution.

**Pull request merged**